model/repository/MonitoringRepository.php

```diff
@@ -460,23 +460,41 @@ protected function prepareOrderStmt($order): string
     {
         $order = explode(',', $order);
         $result = [];
-        $primaryTableColumns = $this->getPrimaryColumns();
-        foreach ($order as $ruleNum => $orderRule) {
-            preg_match('/([a-zA-Z_][a-zA-Z0-9_]*)\s?(asc|desc)?\s?(string|numeric)?/i', $orderRule, $ruleParts);
+        foreach ($order as $orderRule) {
+            $result[] = $this->buildSingleOrderRule($orderRule);
+        }
 
-            if (!in_array($ruleParts[1], $primaryTableColumns)) {
-                $colName = $ruleParts[1];
-                $this->queryParams[] = $colName;
+        return implode(', ', $result);
+    }
+
+    private function buildSingleOrderRule(string $orderRule): string
+    {
+        preg_match(
+            '/([a-zA-Z_][a-zA-Z0-9_]*)\s?(asc|desc)?\s?(string|numeric)?/i',
+            $orderRule,
+            $ruleParts
+        );
```

Comment:
Could you please extract the matches into explicitly-named variables to improve the readability of the code below?
```suggestion
            $ruleParts
        );

        $orderBy = $ruleParts[1];
        $order = $ruleParts[2] ?? 'ASC';
        $type = $ruleParts[3] ?? null;
```